docs/connector-development/connector-builder-ui/overview.md

```diff
@@ -1,56 +1,41 @@
 
 Follow [the tutorial](./tutorial.mdx) for an example of what this looks like in practice.
 :::
 
-## Exporting the connector
-
-:::info
-If you choose to contribute your connector to the Airbyte connector catalog, making it publicly available outside of your workspace, you'll need to export it and go through the process of submitting it for review. 
-:::
-
-Connector Builder leverages the [low-code CDK](https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/yaml-overview) under the hood, turning all configurations into the YAML format. Typically, it's not necessary to interact with the YAML representation. However, you can export the connector YAML into a file and build a docker image containing the connector which can be shared more widely:
-
-1. Use Connector Builder to iterate on your low-code connector
-2. Export the YAML into a low-code connector module on your local machine
-3. Build the connector's Docker image
-4. Use the built connector image in Airbyte
-
-Once you're done iterating on your connector in the UI, you'll need to export the low-code YAML representation of the connector to your local filesystem into a connector module. This YAML can be downloaded by clicking the `Download Config` button in the bottom-left.
-
-Create a low-code connector module using the connector generator (see [this YAML tutorial for an example](../config-based/tutorial/1-create-source.md)) using the name you'd like to use for your connector. After creating the connector, overwrite the contents of `airbyte-integrations/connectors/source-<connector name>/source_<connector name>/manifest.yaml` with the YAML you created in the UI.
+## Contributing the connector
 
-Follow the instructions in the connector README to build the Docker image. Typically this will be something like `docker build . -t airbyte/source-<name>:<version>`.
+If you'd like to share your connector with other Airbyte users, you can contribute it to Airbyte's GitHub repository right from the Builder.
 
-From this point on your connector is a regular low-code CDK connector. It can now be distributed as a docker image and be made part of the regular Airbyte connector catalog. For more information, read the [overview page for the publishing process](/connector-development/#publishing-a-connector).
+1. Click "Publish" chevron -> "Contribute to Marketplace"
+2. Fill out the form: add the connector description, and provide your GitHub PAT (Personal Access Token) to create a pull request
+3. Click "Contribute" to submit the connector to the Airbyte catalog
 
-:::note
-Connector Builder UI is in beta, which means it’s still in active development and may include backward-incompatible changes. Share feedback and requests with us on our Slack channel or email us at feedback@airbyte.io
+Reviews typically take under a week.
```

Comment:
are we committing to this timeline?

Reply:
Not entirely true — aspirational, but I would be happy if anyone asks me and cites this doc.